docs/installation/base-url.md

```diff
@@ -0,0 +1,46 @@
+# Add ability to run Headlamp with a base-url like "/headlamp"
+
+So can use Headlamp at `http://example.com/headlamp` rather than at `http://headlamp.example.com/`.
```

Comment:
I think this needs a bit more context as the example `headlamp.example.com` may lead to doubts IMO.

Reply:
This is done.